helm/hpcc/templates/dali.yaml

```diff
@@ -78,7 +86,7 @@ data:
     dali:
 {{ toYaml (omit $dali "logging" "services") | indent 6 }}
 {{- include "hpcc.generateLoggingConfig" $commonCtx | indent 6 }}
-{{ include "hpcc.generateVaultConfig" (dict "root" $ "categories" (list "system" ) ) | indent 6 }}
+{{ include "hpcc.generateVaultConfig" (dict "root" $ "categories" $secretsCategories ) | indent 6 }}
```

Comment:
related to other comment re. var nameing, will be clearer if renamed daliSecretsCategoreis